Pandas_and_visualization_hw.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[102]:
+
+
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tck
+import seaborn as sns
+
+
+# In[2]:
+
+
+os.chdir("/home/ministreliya/IB/hw_pandas_vis/")
+os.listdir()
+
+
+# In[3]:
+
+
+get_ipython().system('unzip diffexpr_data.tsv.zip')
+
+
+# # Task 1
+
+# In[73]:
+
+
+def read_gff(path_to_file):
+    gff = pd.read_csv(path_to_file,
+                      sep = "\t",
+                      header = 0,
```

Comment:
Это не универсальный способ удаления комментов, он работает только если они занимают 1 строку
```suggestion
                      comment="#",
```